x-pack/plugins/reporting/export_types/common/execute_job/get_conditional_headers.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { ConditionalHeaders, ConfigObject, KbnServer, ReportingJob } from '../../../types';
+
+export const getConditionalHeaders = ({
+  job,
```

Comment:
Same comment about the `job` param here as previous. It's not used in this file, so the benefit of having it is for the consumer, which is an outside detail

Reply:
same issue as above. If there is a simple way to not pass these unused parameters but still have the future functions in the pipe work then I think thats the way to go. Let me know if there is a way to do this.

Reply:
I don't know about this code in enough depth to think of alternatives for now. I appreciate you looking into it